ssl/t1_trce.c

```diff
@@ -1347,7 +1364,7 @@ static int ssl_print_ticket(BIO *bio, int indent, SSL *s,
     }
     if (!ssl_print_hexbuf(bio, indent + 2, "ticket", 2, &msg, &msglen))
         return 0;
-    if (SSL_IS_TLS13(s) && !ssl_print_extensions(bio, indent + 2, 0,
+    if (SSL_IS_TLS13(ssl) && !ssl_print_extensions(bio, indent + 2, 0,
```

Comment:
since you changed this line, can you break before the ```&&&``` and rewrap the second func call? Thanks.

Reply:
Fixed.
Run the format-util to fix a few more  format nits .